### PR TITLE
add test file

### PR DESCRIPTION
**Pull request by Maxvgrad**

### CODE REVIEW

File: feature.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score
+import unittest
+
+def func(a, b, c=0.25, d=42):
+    X = a.drop(b, axis=1)
+    y = a[b]
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=c, random_state=d)
```
Comment (Maxvgrad) [inline feature.py:10]: Do we have a balanced dataset? Is it make sense to allow enabling stratify configuration?

File: feature.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score
+import unittest
+
+def func(a, b, c=0.25, d=42):
```
Comment (Maxvgrad) [inline feature.py:7]: This function fits data in LogisticRegression model and calculates an accuracy. Shall we rename function to be more explicit?
Also argument names could be more human readable? Shall we rename them as well? 

wdyt?
```suggestion
def fit_and_evaluate(X, prediction_column, dataset_split_ratio=0.25, random_seed=42):
```

File: feature.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    model.fit(X_train, y_train)
+    y_pred = model.predict(X_test)
+    return accuracy_score(y_test, y_pred)
+
+class Test(unittest.TestCase):
+    def test(self):
+        data = {'f1': [0, 1, 2, 1], 'f2': [1, 1, 2, 2], 't': [0, 0, 1, 1]}
+        df = pd.DataFrame(data)
+        result = func(df, 't')
+        self.assertIsInstance(result, float)
```
Comment (Maxvgrad) [inline feature.py:21]: nit: shall we assert that result value is between 0 and 1?

File: feature.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    y = a[b]
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=c, random_state=d)
+    model = LogisticRegression()
+    model.fit(X_train, y_train)
+    y_pred = model.predict(X_test)
+    return accuracy_score(y_test, y_pred)
+
+class Test(unittest.TestCase):
+    def test(self):
+        data = {'f1': [0, 1, 2, 1], 'f2': [1, 1, 2, 2], 't': [0, 0, 1, 1]}
```
Comment (Maxvgrad) [inline feature.py:18]: Shall we add a test on a real data?
